src/main/scala/org/graphframes/lib/ShortestPaths.scala

```diff
@@ -58,21 +58,16 @@ class ShortestPaths private[graphframes] (private val graph: GraphFrame) extends
   }
 }
 
-/**
- * Computes shortest paths to the given set of landmark vertices, returning a graph where each
- * vertex attribute is a map containing the shortest-path distance to each reachable landmark.
- */
 private object ShortestPaths {
 
   private def run(graph: GraphFrame, landmarks: Seq[Any]): GraphFrame = {
-    val longLandmarks = landmarks.map(PageRank.integralId(graph, _))
+    val longLandmarks = landmarks.map(GraphXConversions.integralId(graph, _))
     val gx = graphxlib.ShortestPaths.run(
       graph.cachedTopologyGraphX,
       longLandmarks).mapVertices { case (_, m) => m.toSeq }
     GraphXConversions.fromGraphX(graph, gx, vertexNames = Seq(DISTANCE_ID))
   }
 
-  private val DISTANCE_ID = "distance"
+  private val DISTANCE_ID = "distances"
```

Comment:
That one is stated in the doc.  But yeah, I'm not sure whether we want to encourage users to refer to these vals or to Strings.  I guess strings are easier in general, but SVDPlusPlus is an exception since it's a really weird and unclear API.